Makefile

```diff
@@ -14,10 +14,15 @@ RELEASE_FILE :=$(shell cat package.json | jq -r .name)-$(shell cat package.json
 .PHONY: build-release
 build-release: 
 	npm run build
-	rm -rf releases
-	mkdir -p releases
+	cp package.json dist/
+	npm run sign
+
+	mv dist/ servicenow-cloud-observability-datasource
 	
-	ln -s dist servicenow-cloud-observability-datasource
+	mkdir -p releases
 	zip -r "releases/$(RELEASE_FILE)" servicenow-cloud-observability-datasource
-	rm servicenow-cloud-observability-datasource
+	rm -rf servicenow-cloud-observability-datasource
+
+.PHONY: build-md5
+build-md5:
 	echo "$(shell md5 < releases/$(RELEASE_FILE))  $(RELEASE_FILE)" > releases/$(RELEASE_FILE).md5
```

Comment:
Running this as part of the build-release command is resulting in an empty md5 hash with the error: `/bin/sh: releases/servicenow-cloud-observability-datasource-1.0.0.zip: No such file or directory` 🤔 

It looks like the shell command can't find the not-yet-created file.

Calling this as a subsequent step successfully outputs the hash.